server/handler.go

```diff
@@ -304,7 +304,7 @@ func (h *Handler) RemoveOperator(regionID uint64) error {
 		return ErrOperatorNotFound
 	}
 
-	c.opController.RemoveOperator(op)
+	_ = c.opController.RemoveOperator(op)
```

Comment:
How about this one, should we handle it?

Reply:
I think the aim of `Handle.RemoveOperator` is to remove the `op`. If this one returns false, the `op` must be removed by another thread. So, the aim will be achieved no matter what is returned.

Reply:
It seems that we can remove all `_` when we don't need to handle it.

Reply:
I'd like to use the `_` to indicate that the return value is ignored intentionally.